tests/fs/test_mmap.c

```diff
@@ -176,5 +176,53 @@ int main() {
     printf("/yolo/sharedoffset.txt content=%s %d\n", buffer + offset, offset);
     fclose(fd);
   }
+
+  /**
+   * MMAP to and expanded file
+   * 
+   * When appending to a file, the buffer is expanded in chunks, and so the actual length
+   * of the file could be less than the buffer length.
+   * 
+   * When using mmap for an expanded file, we have to make sure that content from the buffer
+   * is not written beyond the allocated memory area for the mmap operation.
+   */
+  {
+    const char* path = "/yolo/expandedfile.txt";
+
+    int fd = open(path, O_RDWR | O_CREAT, (mode_t)0600);
```

Comment:
Just inline path since its only used once.

Reply:
should be ok now